dmoj/result.py

```diff
@@ -1,8 +1,29 @@
+import os
+import signal
 import six
 
 from dmoj.utils.unicode import utf8text
 
 
+def set_result_flag(process, result):
```

Comment:
Why not make this a method on `Result`? It's clearly associated with that class...